app/src/utils/data/grants.ts

```diff
@@ -53,11 +53,14 @@ export async function getAllGrants(forceRefresh = false) {
                   owner
                   payee
                   metaPtr
+                  createdAtTimestamp
+                  lastUpdatedTimestamp
                   lastUpdatedBlockNumber
                 }
               }`,
               fromBlock
             );
+            console.log(grants);
```

Comment:
do we need to log here? if so, I'd maybe make it more descriptive; or remove it otherwise

Reply:
Noo, that was accidentally left behind - I'll get it removed 😅 👍